### PR TITLE
Style team and user pages

### PR DESCRIPTION
**Pull request by chrisrhoton**
This updates the team and user views to a more consistent styling.  That doesn't mean they're pretty, just that they're consistent.

Note: there are some differences in the styling of forms on the team and user pages from those on the library and module pages.  I'm going to update the library and module pages in a separate commit.

Please review the team pages as both a student and admin, the user create page (#/admin/user/create) and your individual student profile (along with the edit page).

### CODE REVIEW

File: assets/js/templates/directives/admin_module_template.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <div class="row row-bottom-margin">
     <div class="col-xs-4">
-     <h2>Module View</h2>
+      <h2>Module View</h2>
```
Comment (xoddong) [inline assets/js/templates/directives/admin_module_template.html]: space

**Comment by tesk9** [comment]
With the caveat that the add admin button not end up on the top line along with the cohort header, because it would make me sad, this does serve to make the styling consistent. Yay consistency.

**Comment by xoddong** [comment]
Looks good!

File: assets/js/templates/directives/admin_module_template.html (hunk 1 above)
Comment (sdevani) [inline assets/js/templates/directives/admin_module_template.html]: This space is actually good because it corrects the previously incorrect spacing

**Pull request merged**